sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchResponseParser.java

```diff
@@ -105,31 +108,25 @@ private static CosmosBatchResponse populateFromResponseContent(
         final boolean shouldPromoteOperationStatus) {
 
         final List<CosmosBatchOperationResult> results = new ArrayList<>(request.getOperations().size());
-        final byte[] responseContent = documentServiceResponse.getResponseBodyAsByteArray();
-
-        if (responseContent[0] != (byte)HYBRID_V1) {
-            // Read from a json response body. To enable hybrid row just complete the else part
-            final ObjectMapper mapper = Utils.getSimpleObjectMapper();
-
-            try {
-                final List<CosmosItemOperation> cosmosItemOperations = request.getOperations();
-                final ObjectNode[] objectNodes = mapper.readValue(responseContent, ObjectNode[].class);
+        final ArrayNode responseContent = (ArrayNode)documentServiceResponse.getResponseBody();
+
+        // Read from a json response body. To enable hybrid row just complete the else part
```

Comment:
we removed the hybrid row checking here? is that intentional?

Reply:
Yes - we don't support hybrid row anyway and there is no plan to add it